LibraryAccess/src/shared/IconTitle.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+export const ICONS = {
```

Comment:
Putting these in an object mostly just makes it easier to export/import, and then someone can look at this file and know what they have available to them.